backend/src/routes/v2/model/patchModel.ts

```diff
@@ -18,6 +18,7 @@ export const patchModelSchema = z.object({
     settings: z
       .object({
         ungovernedAccess: z.boolean().optional().openapi({ example: true }),
+        allowTemplating: z.boolean().optional().openapi({ example: true }),
```

Comment:
This will also need adding to the zod schema in `backend/src/routes/v2/model/postModel.ts`